server/parsec/asgi/rpc.py

```diff
@@ -54,6 +54,32 @@
 logger = get_logger()
 
 
+def block_repr(block: bytes) -> str:
+    if len(block) <= 64:
+        return repr(block)
+    return f"{block[:64]}... ({len(block)} bytes)"
```

Comment:
Maybe make it more explicit?
```suggestion
MAX_LOGGED_BLOCK_SIZE = 64

def block_repr(block: bytes) -> str:
    """Truncate block repr to save space in logs"""
    if len(block) <= MAX_LOGGED_BLOCK_SIZE:
        return repr(block)
    return f"{block[:MAX_LOGGED_BLOCK_SIZE]}... ({len(block)} bytes)"
```